kratos/utilities/quaternion.h

```diff
@@ -432,6 +433,43 @@ namespace Kratos
 			return result;
 		}
 
+		/**
+		Returns a Quaternion that represents a rotation between two given normalized vectors
+		@param a the first vector
+		@param b the second vector
+		@param radians the rotation angle in radians
+		@return a Quaternion that represents a rotation between two given normalized vectors
+		@brief see https://stackoverflow.com/questions/1171849/finding-quaternion-representing-the-rotation-from-one-vector-to-another
+		@
+		*/
+		template<class TVector3>
+		static inline Quaternion FromTwoVectors(TVector3 a, TVector3 b, const double Tolerance = std::numeric_limits<double>::epsilon())

```

Comment:
as this is core can you please make a separate PR and add a test?

Reply:
Since is not used in this PR I'll erase the changes